### DISCUSSION TITLE
Add French as Additional default language

### DISCUSSION THREAD

**Discussion by chusla** [Web3Auth/Web3Auth#562] [Feature Requests]
Hi team, love the product so far.  I have a client who is targeting France and other french speaking customers.  Unfortunately, french is not currently a default language supported by web3auth.  Is it possible to add this language or alternatively a way for users to modify language directly?  

or perhaps use something like weglot to translate on the fly?

Open to all options. the dapp is really coming together but english login screens are jarring at best and could confuse customers at worst.  Thanks!

cc'ing @yashovardhan

**Comment by odepiesse** [comment]
+1

**Comment by shahbaz17** [comment]
Hey @chusla @odepiesse

We're in the process of adding `French` and other requested languages.

**Reply by chusla** [reply]
Heck yeah!  Any idea on timing?

**Comment by odepiesse** [comment]
I though it was included in V3 but it's not :/ 
@shahbaz17, do you have any timeline on release ?

**Reply by odepiesse** [reply]
I think I will send you a PR soon

**Comment by nisa10880** [comment]
+1
